Source/AliveLibAE/Sys.cpp

```diff
@@ -1022,7 +1023,30 @@ EXPORT s8 CC Sys_PumpMessages_4EE4F4()
     SDL_Event event;
     while (SDL_PollEvent(&event))
     {
-        // Pump events
+#if ORIGINAL_PS1_BEHAVIOR // OG Change - Automatically determines active controller (gamepad/keyboard)
```

Comment:
I don't think this should all be using ORIGINAL_PS1_BEHAVIOR? Probably another option called auto switch to game pad or something?